src/SFA.DAS.ApplyService.InternalApi.Types/SignInCallback.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace SFA.DAS.ApplyService.InternalApi.Types
 {
-    public class DfeSignInCallback
+    public class SignInCallback
```

Comment:
I was previously thinking that Dfe was being left all over the code in case we migrated back to the Dfe service?

Reply:
LOL. If we do migrate back, then SignInCallback is still valid.

Reply:
And after all the work put into ASLogin, we're not going back to DfE. 😉

Reply:
I like it being generic. I guess if we have multiple SignIn systems then make it abstract at that point

Reply:
I think a Tech Debt needs raising then to remove all mentions of Dfe Sign in - some of the log messages you added mention it in the new code as well?